cmake/dependencies/ABLASTR.cmake

```diff
@@ -178,7 +178,7 @@ set(ImpactX_openpmd_src ""
 set(ImpactX_ablastr_repo "https://github.com/ECP-WarpX/WarpX.git"
     CACHE STRING
     "Repository URI to pull and build ABLASTR from if(ImpactX_ablastr_internal)")
-set(ImpactX_ablastr_branch "24.07"
+set(ImpactX_ablastr_branch "b58343a9cf31ad2e2e0c47b2c5f0b15cc122a961"
```

Comment:
What is the origin of the string on line 181?

Reply:
That is the hash of the latest commit in WarpX:
- https://github.com/ECP-WarpX/WarpX/commits/development/
- https://github.com/ECP-WarpX/WarpX/commit/b58343a9cf31ad2e2e0c47b2c5f0b15cc122a961

We usually use tags, aka aliases for a commit, but since we are between releases I used a specific commit in `development`.